components/esm/loadland.hpp

```diff
@@ -110,7 +110,7 @@ struct Land
         static void transposeTextureData(const uint16_t *in, uint16_t *out);
     };
 
-    void load(ESMReader &esm, bool &isDeleted);
+    virtual void load(ESMReader &esm, bool &isDeleted);
```

Comment:
Yes, but OpenCS <i>knows</i> that it is dealing with the derived class, so the dynamic dispatch should not be necessary. And as for the game itself, there is no reason for the function to be virtual since it's not being overridden there.